opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -168,21 +169,22 @@ class Measure(Metric):
     Measure metrics represent raw statistics that are recorded.
     """
 
-    def get_handle(self, label_set: LabelSet) -> "MeasureHandle":
-        """Gets a `MeasureHandle` with a float value."""
-        return MeasureHandle()
+    def bind(self, label_set: LabelSet) -> "BoundMeasure":
+        """Gets a `BoundMeasure` with a float value."""
```

Comment:
I know it's not strictly related with this PR but:

```suggestion
        """Gets a `BoundMeasure`."""
```